src/Files.App/Views/Properties/SecurityPage.xaml

```diff
@@ -368,6 +421,18 @@
 						IsChecked="{x:Bind SecurityViewModel.SelectedAccessControlEntry.DeniedReadAccess, Mode=TwoWay}"
 						IsEnabled="{x:Bind SecurityViewModel.SelectedAccessControlEntry.DeniedInheritedReadAccess, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}" />
 
+					<!--  (Alternal Color)  -->
```

Comment:
What is this for?

Reply:
![image](https://user-images.githubusercontent.com/62196528/232265692-174135b1-60b8-4261-9c65-8c4b20588ab2.png)

Reply:
Can you revert this change? It doesn't align with the expected design.

Reply:
Reverted